build_tools/ci/cpu_comparison/run.py

```diff
@@ -616,7 +620,7 @@ def run(self, config):
                 config,
                 test_files_dir / f"{name}.mlir",
                 tile_pipeline="conv-decompose",
-                lower_to_aie_pipeline="air",
+                lower_to_aie_pipeline="objectFifo",
```

Comment:
@erwei-xilinx I think all of these tests fail with the air pipeline with the changes to tiling introduced in this PR.